### ISSUE TITLE
Gradient Clipping

### ISSUE THREAD

**Issue by unlugi** [ustclby/Unsupervised-Domain-Specific-Deblurring#1]
Dear author,

In the DRIT model you have taken inspiration from, gradient clipping is applied to stabilize gan training. I couldn't see that in your implementation if I am not mistaken. I was wondering if this is intentional.

Thank you.

**Comment by unlugi** [comment]
Also,
I am training with a different dataset. And after a while, the generator and discriminator losses blow up, and I start to receive nan values. What are your recommendations for adapting the training process for a new dataset?

**Comment by zabir-nabil** [comment]
@unlugi how did you solve the loss issue can you tell?

**Comment by liuzhuang1024** [comment]
I also encountered the same problem. No matter how you do it, there are situations where loss is equal to nan. I don’t know if the author has encountered it during the training.
